src/configuration/configuration_manager.hpp

```diff
@@ -57,8 +57,8 @@ class configuration_manager {
 protected:
     void remove_config(const configuration_change_spec &cfg);
 
-    static void load(
-        raw_configuration::map &root, configuration_collector &collector, base_ruleset_info &info);
+    static bool load(
```

Comment:
document what the return value means. It's not clear it's used to signal errors or more generally whether something was loaded (from the impl it seems it returns false if there are errors only).